addon/components/learnergroup-tree.js

```diff
@@ -54,4 +54,8 @@ export default class LearnergroupTree extends Component {
       this.args.add(learnerGroup);
     }
   }
+
+  sortByTitle(learnerGroupA, learnerGroupB) {
+    return learnerGroupA.title.localeCompare(learnerGroupB.title, undefined, {numeric: true});
```

Comment:
Instead of undefined can you pass the current locale in [like this](https://github.com/ilios/common/blob/7ca9d7ea26810af47af90a19702c709f8328a9b1/addon/components/user-search.js#L126-L131) it may do the same thing by reading the locale from the html language attribute, but I like the explicit one better.